packages/lavapack/src/pack.js

```diff
@@ -15,6 +15,7 @@ const assert = require('assert')
 const JSONStream = require('JSONStream')
 const through = require('through2')
 const umd = require('umd')
+// eslint-disable-next-line node/prefer-global/buffer
```

Comment:
Is there some subtlety making the `require('buffer')` necessary? If not it seems better to just remove it.